timeline/views_ajax.py

```diff
@@ -132,20 +149,24 @@ def _html_print_array(self, array, str_list, indent):
         str_list.append('&nbsp;&nbsp;' * (indent - 1))
         str_list.append('<span class="json-close-bracket">]</span>')
 
-    def get_event_extra(self, event):
+    def get_event_extra(self, event, status=None):
```

Comment:
I added extra data to a project event's status update, but for some reason it is showing up empty in the view. Please check:
![2023-01_timeline_extra_data_bug](https://user-images.githubusercontent.com/11211145/214099089-8a7babe0-fb27-40a1-b9b3-506d37a796cc.png)

Reply:
That's weird, by me it's functionating just ok
![Screenshot from 2023-01-26 11-25-08](https://user-images.githubusercontent.com/86564271/214813219-73c91e08-5be1-4532-b3ea-44f4aa4f2b79.png)
![Screenshot from 2023-01-26 11-24-47](https://user-images.githubusercontent.com/86564271/214813319-afa0ae53-505a-4b91-bb55-0f4fc41ce90e.png)

Reply:
I replicated the event where this fails. When browsed in the UI, the extra data for the status appears empty even though it is `{'test': 'something'}`.

```
from projectroles.plugins import get_backend_api
timeline = get_backend_api('timeline_backend')
User = get_user_model()
u = User.objects.first()
p = Project.objects.filter(type='PROJECT').first()
e = timeline.add_event(project=p, app_name='filesfolders', user=u, event_name='folder_create', description='create folder', extra_data={'test': 'something'})
s = e.set_status('OK', 'All OK', {'test': 'something'})
```

Reply:
After the most recent updates, I no longer see the link for extra data for the status. But the extra data is there, I even verified to make sure in Django shell. My immediate guess is that something goes wrong when you create the link for the Ajax view.

Reply:
It's really strange, because it works by me)
![Screenshot from 2023-01-30 14-31-49](https://user-images.githubusercontent.com/86564271/215491633-d60ab62a-c10d-4e7d-906d-a6b72e761d1f.png)

Reply:
This works now. The last failure was most likely some browser cache issue on my end..